package.json

```
@@ -16,7 +16,7 @@
     "url": "https://github.com/M6Web/superagent-mock"
   },
   "peerDependencies": {
-    "superagent": "^3.6.0"
+    "superagent": ">=3.6.0"
```

Maybe because I'm not sure it is compatible with superagent v5
```suggestion
    "superagent": ">=3.6.0 <5.0.0"
```

---

We upgraded from `"superagent": "^3.8.1"` to `"superagent": "^5.1.2"` (v5.2.1 installed) on December 19 2019 and have not noticed any issues. Of course, it may be that we are not using all the functionality hence did not find any issues. On the other hand, if most users are using v5.x.x of superagent, then the warning will still appear.